lib/api/authn/force-user.js

```diff
@@ -0,0 +1,19 @@
+const debug = require('../../debug').authentication
+
+/**
+ * Enforces the `--force-user` server flag, hardcoding a webid for all requests,
+ * for testing purposes.
+ */
+function initialize (app, argv) {
+  const forceUserId = argv.forceUser
+  app.use('/', (req, res, next) => {
+    debug(`Identified user (override): ${forceUserId}`)
+    req.session.userId = forceUserId
+    res.set('User', forceUserId)
```

Comment:
ahh, you're right. so then maybe we should wrap setting that header in an auth method check (only do it if authMethod is tls)